layouts/index.html

```diff
@@ -5,46 +5,74 @@
         {{ partial "skip.html" . }}
         {{ partial "nav.html" . }}
```

Comment:
On the homepage, consider pulling the nav. it's not really needed on this page and will distract from the rest of the content on the page.